FateGrandAutomata/SupportImgNamerActivity.cs

```diff
@@ -199,10 +199,10 @@ bool Validate(string OldPath, string NewFileName, Regex Regex, string InvalidMsg
         const string InvalidCharsMsg = "'<, >, \", |, :, *, ?, \\, /'";
```

Comment:
Last thing, I would remove the `'` in this message. Otherwise people might think that `'` is included in the character list